Pages/rules.css

```diff
@@ -0,0 +1,79 @@
+@import url(../UtilityCSS/utility.css);
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+:root {
+  --background-color: #111827;
+  --card-background: #19212c;
+  --margin-sm: 4px 6px;
+  --grey-color: #94a3b8;
+  --text-color: #fafafa;
+  --medium-color: #d97706;
+  --hard-color: #dc2626;
+  --active-color: #eab308;
+  --correct-color: #4caf50;
+  --wrong-color: #f44336;
+}
+
+.nav-title {
+  font-size: 4rem;
+  align-self: center;
+  margin: var(--margin-sm);
+}
+
+body {
+  background-color: var(--background-color);
+  color: var(--text-color);
+}
+
+.nav-link {
+  text-decoration: none;
+  color: var(--text-color);
+  font-size: 2rem;
+  padding: 1rem;
+}
+
+.border-solid {
+  border: 1px solid #64748b;
+}
+
+.active {
+  color: var(--active-color);
+}
+
+.rule-box {
+  margin: 2rem;
+  height: 30rem;
+  width: 50%;
+}
+
+.rule {
+  align-self: center;
+  font-size: 3rem;
+  margin: var(--margin-sm);
+  color: var(--grey-color);
+}
+
+.quiz-start-btn {
+  height: 5rem;
+  width: 35rem;
+  border: none;
+  outline: none;

```

Comment:
Don't give `outline:none` which affects **accessibility**. Instead you can use 

           outline: 2px solid transparent;
           outline-offset: 2px;

Reply:
didnt knew that will implement this with other suggested feedbacks